### PR TITLE
docs: example of dynamic row action in table component

### PR DESCRIPTION
**Pull request by amontalvof**
fix: #1815 

## Changes proposed in this PR:
-

- [ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/nexxtway/react-rainbow/blob/master/CONTRIBUTING.md#submitting-a-pull-request).

@nexxtway/react-rainbow

### CODE REVIEW

File: src/components/Table/readme.md [hunk 1]
```diff
@@ -577,6 +577,114 @@ const StatusBadge = ({ value }) => <Badge label={value} variant="lightest" style
...
+        id: '5678asdfgh',
+    },
+    {
+        name: 'Carlos',
+        company: 'Oracle',
+        email: 'carlos@gmail.com',
+        status: 'pending',
+        id: '5678zxcvbn',
+    },
+];
```
Comment (reiniergs) [inline src/components/Table/readme.md:631]: @TahimiLeonBravo I think we are repeating this list on multiple examples, we should have an internal files that export a list if persons that we can use in the examples like:
```
import persons from '/path/to/files';
```

Comment (TahimiLeonBravo) [reply src/components/Table/readme.md:631]: I agree, let's do this @amontalvof

File: src/components/Table/readme.md [hunk 2]
```diff
@@ -577,6 +577,114 @@ const StatusBadge = ({ value }) => <Badge label={value} variant="lightest" style
...
+                icon={<FontAwesomeIcon icon={faEllipsisV} />}
+            />
+        </ButtonGroup>
+    </GlobalHeader>
+    <Table keyField="id" data={data}>
+        <Column header="Name" field="name" />
+        <Column header="Status" field="status" component={StatusBadge} />
+        <Column header="Company" field="company" />
+        <Column header="Email" field="email" />
+        <Column field="status" component={ButtonAction}/>
```
Comment (TahimiLeonBravo) [inline src/components/Table/readme.md:683]: in order to have a more beautiful example we should add a `width={60}` to this column

File: src/components/Table/readme.md [hunk 3]
```diff
@@ -577,6 +577,114 @@ const StatusBadge = ({ value }) => <Badge label={value} variant="lightest" style
...
+    );
+};
+
+const ButtonAction = (props) => {
+    const {value, row:{name}}=props;
+    return (
+        <ButtonMenu
+            id="button-menu-2"
+            menuAlignment="right"
+            menuSize="x-small"
```
Comment (TahimiLeonBravo) [inline src/components/Table/readme.md:658]: we should use the `buttonVariant="base"` on the ButtonMenu component

**Review by TahimiLeonBravo — CHANGES_REQUESTED** [review]

File: src/components/Table/readme.md [hunk 4]
```diff
@@ -577,51 +304,81 @@ const StatusBadge = ({ value }) => <Badge label={value} variant="lightest" style
...
+            <MenuItem label="Edit" onClick={() => console.log(`Edit ${name}`)}/>
+        </>
+    );
+};
+
+const ButtonAction = (props) => {
+    const {value, row:{name}}=props;
+    return (
+        <ButtonMenu
+            id="button-menu-2"
```
Comment (LeandroTorresSicilia) [inline src/components/Table/readme.md:340]: take care this id is not used already in other Table example

File: src/components/Table/exampleData.js [hunk 5]
```diff
@@ -0,0 +1,294 @@
+export const data = [
```
Comment (LeandroTorresSicilia) [inline src/components/Table/exampleData.js:1]: we should move this file to this folder `library/data` where we already have a `user.js` file and it is set globally in the `library/setup.js` file

**Comment by TahimiLeonBravo** [comment]
@amontalvof let's do the new examples in this way:
- Each example we should have a title and a description
- It should look like the design https://zpl.io/29LEldY

File: src/components/Table/readme.md (hunk 4 above)
Comment (amontalvof) [reply src/components/Table/readme.md:340]: I checked and the id is not used in any other example

**Review by LeandroTorresSicilia — APPROVED** [review]

**Pull request merged**